py3status/modules/volume_status.py

```diff
@@ -16,9 +16,12 @@
         (default 0)
     cache_timeout: how often we refresh this module in seconds.
         (default 10)
-    channel: Alsamixer channel to track.
+    channel: Alsamixer channel to track (ignored by pulseaudio)
         (default 'Master')
-    device: Alsamixer device to use.
+    command: Choose between "amixer" and "pamixer". If None, try to guess based
+        on available commands.
+        (default None)
+    device: Device to use.
         (default 'default')
```

Comment:
I think we should change the default to `None`.

My thinking is that as it is `default` for alsa but `0` for pulseaudio that having it as `default` is misleading.  We can then just set it depending on which backend is used like pamixer already does.

The documentation can then be something like  `device: Device to use. Will use default device if none specified.`